src/bttrx_fsm.cpp

```diff
@@ -114,7 +116,13 @@ void BTTRX_FSM::handleStateConfigure()
 	}
 
 	// Enforce configuration
-	string friendly_name = "bt-trx-" + wt32i_.getBDAddressSuffix();
+	string friendly_name = "bt-trx_";
+	String callsign = preferences.getString("callsign", "");
```

Comment:
see comment above

Reply:
This is addressed in commit https://github.com/phl0/firmware/commit/3418d93f7ca2c7895d921a5d942337d344ee3712.